operations/configuration_management/Configuration_Layers.md

```diff
@@ -4,6 +4,60 @@ The Configuration Framework Service \(CFS\) uses configuration layers to specify
 
 Configurations with a single layer are useful when testing out a new configuration on targets, or when configuring system components with one product at a time. To fully configure a node or boot image component with all of the software products required, multiple layers can be used to apply all configurations in a single CFS session. When applying layers in a session, CFS runs through the configuration layers serially in the order specified.
 
+<a name="configuration_layer_example_configuration_single"></a>
+
+### Example Configuration (Single Layer)
+
+The following is an example configuration with a single layer. This can be used
+as a template to create a new configuration JSON file to input to CFS.
+
+```bash
+ncn# cat configuration-single.json
+{
+  "layers": [
+    {
+      "name": "configurations-layer-example-1",
+      "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/example-repo.git",
+      "playbook": "site.yml",
+      "commit": "43ecfa8236bed625b54325ebb70916f55884b3a4"
+    }
+  ]
+}
+```
+
+<a name="configuration_layer_example_configuration_multiple"></a>
+### Example Configuration (Multiple Layers)
+
+The following is an example configuration with multiple layers from one or more
+different configuration repositories. This can be used as a template to create a
+new configuration JSON file to input to CFS.
+
+```bash
+ncn# cat configuration-multiple.json
+{
+  "layers": [
+    {
+      "name": "configurations-layer-example-1",
+      "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/example-repo.git",
+      "playbook": "site.yml",
+      "commit": "43ecfa8236bed625b54325ebb70916f55884b3a4"
+    },
+    {
+      "name": "configurations-layer-example-2",
+      "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/example-repo.git",
+      "playbook": "site-custom.yml",
+      "commit": "43ecfa8236bed625b54325ebb70916f55884b3a4"
```

Comment:
Use a commit hash that is different from the example-1.  Same goes for the cloneUrl (or was it intentional for these to be the same as example-1 and I am not getting it?).